docs/FRAMEWORK-EXAMPLES.md

```diff
@@ -0,0 +1,65 @@
+<!---
+id: framework-examples
+title: Testrunner Toolkit Framework Examples
+sidebar_label: Framework Examples
+--->
+
+# Testrunner Toolkit Framework Examples
+
+The examples here show how Pipeline testing can be used. Try them and find your own use cases. 
+
+Every __testrunner__ image comes with a preconfigured setup that allows you to focus on writing tests instead of tweaking with the configurations. Our initial `testrunner` flavors come either with Puppeteer, Playwright, or TestCafe as an automation framework. 
+
+## Example Snippets
+Each example will start the browser for you and expose the `browser` object ([Puppeteer](https://pptr.dev/#?product=Puppeteer&version=v3.0.3&show=api-class-browser) / [Playwright](https://playwright.dev/#version=v1.0.1&path=docs%2Fcore-concepts.md&q=browser) / [TestCafe](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/browser.html)) to the global scope for you to be accessible in the test:
```

Comment:
> Each example will start the browser for you and expose the `browser` object

This is not correct for Testcafe tests

Reply:
Is TestCafe different in that it serves content via a proxy server/service? Please elaborate and I will change the doc.